lotus-soup/testkit/role_miner.go

```diff
@@ -256,6 +256,12 @@ func PrepareMiner(t *TestEnvironment) (*LotusMiner, error) {
 		return nil, err
 	}
 
+	// set expected seal duration to 1 minute
+	err = n.MinerApi.SectorSetExpectedSealDuration(ctx, 1*time.Minute)
```

Comment:
cc @vyzo this fixes the `deal rejected: cannot seal a sector before 10002` error.